src/Utility/QuickSetup.lua

```diff
@@ -0,0 +1,44 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local QuickSetup = {}
+
+
+function QuickSetup:Run()
```

Comment:
Small nit, would prefer dot index instead of colon here, it's just more inline with the rest of the code.